platforms/Windows/ide/ide.wxs

```diff
@@ -37,6 +37,14 @@
         <File Source="$(TOOLCHAIN_ROOT)\usr\lib\sourcekitdInProc.lib" />
       </Component>
 
+      <Component Directory="_usr_lib">
```

Comment:
This seems like the wrong place ... DLLs are placed into the `bin` directory, `lib` should have static libraries and import libraries.

Reply:
These libraries will be opened by SourceKit using `dlopen` and are not directly linked to SourceKit. Hence, they don’t ned to go into `bin` for Windows to pick them up and I thus thought that it makes more sense to put them in `lib`. Happy to change it, if you prefer to have them in `bin`.

Reply:
I think that they should be in `bin`. Unless they are completely standalone (including any system library dependencies). The current directory is scanned for any dependencies, and having them elsewhere makes that difficult.

Reply:
These libraries are fully standalone and don’t depend on any of the other libraries in `bin`.

Reply:
But that also includes the C/C++ runtimes. They _can_ also be redistributed in the toolchain (and if we run into the cases where we need to, we would fall back to that). In the mean time, note that other "plugin" libraries are loaded from `bin` on Windows.

Reply:
Ah, I see. Moved it to `bin`